enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,256 @@
+  * The `NoConfig` backups relies on `EtcdBackup` CR with `default` name.
+  * The cluster-etcd-operator reacts to the `default` CR by deploying the backup sidecar containers alongside each etcd member. 
+
+
+* Upon enabling the `AutomatedBackup` feature gate, which approach should be used and according to what criteria.
+  * As the `Noconfig` backups is orthogonal to the [automated backup of etcd](https://docs.openshift.com/container-platform/4.15/backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.html#creating-automated-etcd-backups_backup-etcd), it has been decided to use the same feature gate.
+  
+
+* How should the current backup controller distinguish between both approaches.
+  * The `NoConfig` backups relies on `EtcdBackup` CR with `default` name.
+  
+
+* How to disable the `NoConfig` behaviour, if a user want to.
+  * This has not been decided or implemented yet. 
+
+### Implementation Details/Notes/Constraints
+
+The `NoConfig` backups builds atop of the existing [automated backup of etcd](https://docs.openshift.com/container-platform/4.15/backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.html#creating-automated-etcd-backups_backup-etcd).
+
+
+The `PeriodicBackupController` within `cluster-etcd-operator` checks for `EtcdBackup` CR with `default` name. Once found, a sidecar container is deployed within each etcd pod in Openshift cluster.
+The container runs a backup server with the default schedule as defined in default `EtcdBackup` CR. 
+
+The backups are being stored on the local disk of each master node independently, using `hostPath` volume. This replication is required for disaster recovery scenarios where more than one of the master nodes has been lost.
+
+
+A default `EtcdBackup` CR is used for the `NoConfig` backups. A default schedule of daily at midnight has been chosen. See below
+
+```yaml
+apiVersion: config.openshift.io/v1alpha1
+kind: Backup
+metadata:
+  name: default
+spec:
+  etcd:
+    schedule: "0 0 * * *"
+    timeZone: "UTC"
+```
+
+
+
+
+### Risks and Mitigations
+
+## Test Plan
+
+An e2e test will be added to practice the scenario as follows 
+
+- Enable AutomatedBackupNoConfig FeatureGate.
+- Verify that a `Etcdbackup` CR has been installed.
+- Verify that a backup has been taken.
+- Verify that the backups are valid according to the retention policy.
+
+## Graduation Criteria
+TBD
+
+### Dev Preview -> Tech Preview
+TBD
+
+### Tech Preview -> GA
+TBD
+
+### Removing a deprecated feature
+TBD
+
+## Upgrade / Downgrade Strategy
+TBD
+
+## Version Skew Strategy
+TBD
+
+## Operational Aspects of API Extensions
+TBD
+
+#### Failure Modes
+TBD
+
+## Support Procedures
+TBD
+
+## Implementation History
+TBD
+
+### Drawbacks
+
+## Alternatives
+
+**Ideally**, the storage solution to use would have the following characteristics
+- Backups should be taken on a round-robin fashion on each master node to avoid overwhelming a specific etcd member more than others.
```

Comment:
Have we considered the storage space required for the backups, is there configuration for retention? What is the impact of adding this data to the nodes?